ui/brave_ads/BUILD.gn

```diff
@@ -63,13 +64,26 @@ source_set("brave_ads") {
       "padded_image.cc",
       "padded_image.h",
     ]
+
+    if (is_mac) {
+      sources += [ "message_popup_view_mac.mm" ]
+    }
+
+    if (is_win) {
+      sources += [ "message_popup_view_win.mm" ]
```

Comment:
Just curious - `mm` works on Windows?

Reply:
@tmancey You fixed right before :)

Reply:
nah it was a mistake from me :-) the root of all evil, copy & paste!